.github/workflows/sync-docs.yaml

```diff
@@ -0,0 +1,43 @@
+name: Sync files to docs repo
+
+on:
+  push:
+    paths:
+      - tenzir.yaml.example
+      - web/openapi/openapi.yaml
```

Comment:
I'd suggest just always doing this on push. The workflow is cheap to run, and this list is relatively easy to get out of sync.